plugins/browser-plugin-web-vitals/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@snowplow/browser-plugin-web-vitals",
+  "version": "3.10.1",
+  "description": "Adds the capability to track web performance metrics categorized as Web Vitals.",
+  "homepage": "https://github.com/snowplow/snowplow-javascript-tracker",
+  "bugs": "https://github.com/snowplow/snowplow-javascript-tracker/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/snowplow/snowplow-javascript-tracker.git"
+  },
+  "license": "BSD-3-Clause",
+  "author": "Snowplow Analytics Ltd (https://snowplow.io/)",
+  "sideEffects": false,
+  "main": "./dist/index.umd.js",
+  "module": "./dist/index.module.js",
+  "types": "./dist/index.module.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c --silent --failAfterWarnings",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@snowplow/browser-tracker-core": "workspace:*",
+    "@snowplow/tracker-core": "workspace:*",
+    "tslib": "^2.3.1",
+    "web-vitals": "~3.3.2"
```

Comment:
The bundle is not affected since we only use types from the library. The reason for adding this to deps is a practise from https://github.com/microsoft/types-publisher/issues/81#issuecomment-234051338